### PR TITLE
global: fix the license

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: inspirehep/pidstore/__init__.py [hunk 1]
```diff
@@ -1,23 +1,9 @@
+#!/usr/bin/env bash
```
Comment (michamos) [inline inspirehep/pidstore/__init__.py]: This is not a bash script (in every python file).

Comment (drjova) [reply inspirehep/pidstore/__init__.py]: oups my bad thanks

**Comment by drjova** [comment]
@michamos updated

File: inspirehep/pidstore/__init__.py [hunk 2]
```diff
@@ -1,23 +1,8 @@
...
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
-#
-# In applying this license, CERN does not waive the privileges and immunities
-# granted to it by virtue of its status as an Intergovernmental Organization
-# or submit itself to any jurisdiction.
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details..
```
Comment (pazembrz) [inline inspirehep/pidstore/__init__.py]: Don't forget to remove one dot from the end of the sentence.

**Comment by drjova** [comment]
@pazembrz updated!

**Pull request merged**